drivers/storage/scaleio/tests/README.md

```diff
@@ -0,0 +1,31 @@
+# ScaleIO driver testing
```

Comment:
Hi @dvonthenen,

First of all, thank you *so* much for following my example of keeping markdown files to 80 columns wide. I know that Markdown will soft-wrap where necessary, but I find this makes things so much easier to read. Thank you!